app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if "cp " in command:
+        command_ls = command.split()
+        source_file_name = command_ls[1]
+        cp_file_name = command_ls[2]
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names
command, old_file, new_file = command.split()